tests/test_search.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+
+from selenium import webdriver
+from constants import BASE_URL, SEARCH_INPUT, RESULTS_PAGE_TITLE, MAIN_PAGE_TITLE
+from pages.search_page import SearchPage
+from pages.search_results_page import SearchResultPage
+
+
+class SearchTests(unittest.TestCase):
+
+    def setUp(self):
+        self.driver = webdriver.Firefox(executable_path='C:\Program Files\geckodriver\geckodriver')
+        self.driver.get(BASE_URL)
+        self.search_page = SearchPage(self.driver)
+        self.results_page = SearchResultPage(self.driver)
+
+    def test_search_working(self):
+        self.assertTrue('Not on the correct page', self.search_page.is_browser_on_page(MAIN_PAGE_TITLE))
+        self.search_page.search_by_value(SEARCH_INPUT)
```

Comment:
If you are sending the value stored in constants.py, why don't you add it inside the search_by_value method instead of here. 
You normally use this pattern when you have to send a string like:
```
self.search_page.search_by_value("Intel NUC")
```
so that if any other test wants to search for any other thing we can use this method: 
```
self.search_page.search_by_value("Mac book pro")
```